### PR TITLE
Reword Kubernetes overview

### PR DESCRIPTION
**Pull request by YashPimple**
Resolved issue #36689 and made the required changes in the Self-healing  section as per instructed

### CODE REVIEW

**Comment by sftim** [comment]
Hi @YashPimple 

In your own words - I mean, not quoting the original issue - why should we make this change?

(if you're not sure that we should, or about why we should, then that is important context for a reviewer to know)

File: content/en/docs/concepts/overview/_index.md [hunk 1]
```diff
@@ -120,8 +120,10 @@ Kubernetes provides you with:
   containers onto your nodes to make the best use of your resources.
 * **Self-healing**
   Kubernetes restarts containers that fail, replaces containers, kills containers that don't
-  respond to your user-defined health check, and doesn't advertise them to clients until they
-  are ready to serve.
+  respond to your user-defined health check :- does Kubernetses really act on the container 
+  directly or restart the pod which having that container? kubernetes does not replace the
```
Comment (kbhawkey) [inline content/en/docs/concepts/overview/_index.md:124]: Hi @YashPimple.
I'm not clear on whether the changes improve the existing content.

**Comment by divya-mohan0209** [comment]
Hello @YashPimple , gentle reminder. Please advise if you'd be able to continue work on this PR & address feedback by Ritika & Tim? We'll be closing the PR off if we do not receive a response from your side by July 20th, 2023.

**Comment by kbhawkey** [comment]
👋 @YashPimple .
The changes have not seen activity for some time. I plan on closing this pull request, but feel free to reopen.
/close

**Pull request closed without merging**